### PR TITLE
[TW-891] Doc collection run plan changes

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/administration/resource-usage.md [hunk 1]
```diff
@@ -46,6 +47,14 @@ Once you've reached your usage limit, your monitors and scheduled collection run
 
 > Learn more about [viewing and managing your monitor usage](/docs/monitoring-your-api/monitor-usage/).
 
+## Collection runs usage
+
+Your Postman plan gives you a limited number of [collection runs](/docs/running-collections/intro-to-collection-runs/) you can use each month. This limit applies to collections that you run in a workspace or the Scratch Pad using the **Run manually** option (not [scheduled collection runs](/docs/running-collections/scheduling-collection-runs/) in the Postman cloud). Each iteration counts as a single run. You will get a notification in the Collection Runner when you're approaching your usage limit. (Collection run usage doesn't appear on the Resource Usage dashboard.)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/administration/resource-usage.md:52]: What do you think about linking to the Plans page? This way users can learn how many collections they can run per plan type. https://www.postman.com/pricing/

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good! I added one minor suggestion. Thanks!

File: src/pages/docs/administration/resource-usage.md (hunk 1 above)
Comment (HeatherStenson) [inline src/pages/docs/administration/resource-usage.md:52]: Maybe separate out "(not scheduled collection runs in the Postman cloud)" as its own sentence? I kind of got lost in that sentence and worry that other readers might too.

```suggestion
Your Postman plan gives you a limited number of [collection runs](/docs/running-collections/intro-to-collection-runs/) you can use each month. This limit applies to collections that you run in a workspace or the Scratch Pad using the **Run manually** option. This limit doesn't apply to [scheduled collection runs](/docs/running-collections/scheduling-collection-runs/) in the Postman cloud. Each iteration counts as a single run. You will get a notification in the Collection Runner when you're approaching your usage limit. (Collection run usage doesn't appear on the Resource Usage dashboard.)
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! Just one small thought about wording.

File: src/pages/docs/administration/resource-usage.md (hunk 1 above)
Comment (postfrzero) [inline src/pages/docs/administration/resource-usage.md:52]: > Each iteration counts as a single run.

A single run can have multiple iterations and it'll still count as a single run.

> You will get a notification in the Collection Runner when you're approaching your usage limit.

They won't get a notification in the classical sense, but instead messaging will appear within the collection runner to let them know. I'm not how we can tweak the wording here to communicate that. 😅

**Review by postfrzero — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/resource-usage.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/resource-usage.md]: Sounds good! There was a link to the pricing page at the top of this page, but I went ahead and added it here as well, and in each of the other sections.

Comment (mark-anthony-postman) [reply src/pages/docs/administration/resource-usage.md]: Great, thanks for clarifying both of these points! I updated the text to (hopefully) be more accurate 😀

**Review by postfrzero — APPROVED** [review]

**Pull request merged**